include/databroker/types/topic/WildcardTopic.hpp

```diff
@@ -34,6 +34,9 @@ class WildcardTopic : public AbstractTopic
     //! Using parent constructos
     using AbstractTopic::AbstractTopic;
 
+    //! Constructor that allows any type
+    WildcardTopic(const std::string& topic_name);
```

Comment:
Missing test for this function.

Reply:
Just add some new cases to the topic tests as that PR is already merged.

Reply:
:ok: